pom.xml

```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
is it a multi module project?

Reply:
I've asked Aliaksei to implement the project using a single-module structure.